static/app/components/superuserStaffAccessForm.tsx

```diff
@@ -59,9 +59,11 @@ class SuperuserStaffAccessForm extends Component<Props, State> {
       return;
     }
 
-    await this.getAuthenticators();
+    const authenticators = await this.getAuthenticators();
+    this.setState({authenticators: authenticators});
```

Comment:
I don't see this state being modified anywhere else. Does it even need to be state?

Reply:
Would removing it look like this? https://github.com/getsentry/sentry/pull/70365/